File: include/extras-kconfigwidgets/libkstylemanager.zig
const QtC = @import("qt6zig");
const qtc = @import("qt6c");

/// https://api-staging.kde.org/kstylemanager.html
pub const kstylemanager = struct {
    /// [Qt documentation](https://api-staging.kde.org/kstylemanager.html#initStyle)
    ///
    ///
    pub fn InitStyle() void {
        qtc.KStyleManager_InitStyle();
    }

    /// [Qt documentation](https://api-staging.kde.org/kstylemanager.html#createConfigureAction)
    ///
    /// ``` param1: QtC.QObject ```
    pub fn CreateConfigureAction(param1: ?*anyopaque) QtC.QAction {
        return qtc.KStyleManager_CreateConfigureAction(@ptrCast(param1));
    }
};

File: src_original/extras-kconfigwidgets/libkstylemanager.cpp
#include <KStyleManager>
#include <QAction>
#include <QObject>
#include <kstylemanager.h>
#include "libkstylemanager.h"
#include "libkstylemanager.hxx"

void KStyleManager_InitStyle() {
    KStyleManager::initStyle();
}

QAction* KStyleManager_CreateConfigureAction(QObject* param1) {
    return KStyleManager::createConfigureAction(param1);
}
